build/tutorials/exchanges/exchange-c-chain-integration.md

```diff
@@ -0,0 +1,70 @@
+# Avalanche C-Chain Integration Guide
+
+## Overview
+
+The objective of this document is to provide a brief overview of how to integrate with the EVM-Compatible Avalanche C-Chain. For teams that already support ETH, supporting the C-Chain is as straightforward as spinning up an Avalanche node (which has the same API as go-ethereum) and populating Avalanche’s ChainID (43114) when constructing transactions.
+
+### Running Avalanche node
+
+If you want to build your node form source or include it in a docker image, reference the [AvalancheGo GitHub repository](https://github.com/ava-labs/avalanchego). To quickly get up and running, you can use the [node installation script](../nodes-and-staking/set-up-node-with-installer.md) that automates installing and updating avalanchego node as a systemd service on Linux, using prebuilt binaries.
+
+### Configuring Avalanche node
+
+All command line options available are described [here](../../references/command-line-interface.md), along with their default values.
+
+You can supply options on the command line, or use the config file, which is probably easier to work with. Use `—config-file=config.json` option, and then provide complete configuration in the `config.json` file, for example:
```

Comment:
"which is probably easier to work with" -> "which can be easier to work with when a lot of custom options are configured"